src/Organization/schema.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class Organization_type(Enum):
```

Comment:
Can use Camel letter
```suggestion
class OrganizationType(Enum):
```